__tests__/ExpensiMark-HTML-test.js

```diff
@@ -1491,3 +1490,91 @@ test('Mention', () => {
     testString = '@user@DOMAIN.com';
     expect(parser.replace(testString)).toBe('<mention-user>@user@DOMAIN.com</mention-user>');
 });
+
+describe('edit mode', () => {
+    test('quote without space', () => {
+        const quoteTestStartString = '>Hello world';
+        const quoteTestReplacedString = '<blockquote>Hello world</blockquote>';
+
+        expect(parser.replace(quoteTestStartString, {shouldKeepWhitespace: true})).toBe(quoteTestReplacedString);
+    });
+
+    test('quote with space', () => {
```

Comment:
Nothing important but let's emphasize the fact that here we have exactly one space.
```suggestion
    test('quote with single space', () => {
```